invariant.ts

```diff
@@ -254,15 +277,27 @@ export const checkInvariants = (
           radio.emit(
             "logMessage",
             red(
-              `${invariantCallerWallet} [FAIL] ${getContractNameFromRendezvousId(
-                r.rendezvousContractId
-              )} ${underline(r.selectedInvariant.name)} ${printedInvariantArgs}`
+              `₿ ${simnet.burnBlockHeight.toString().padStart(6)} ` +
+                `Ӿ ${simnet.blockHeight.toString().padStart(6)}   ` +
+                `${invariantCallerWallet} ` +
+                `[FAIL] ` +
+                `${getContractNameFromRendezvousId(r.rendezvousContractId)} ` +
+                `${underline(r.selectedInvariant.name)} ` +
+                printedInvariantArgs
             )
           );
 
           // Re-throw the error for fast-check to catch and process.
           throw error;
         }
+
+        try {
+          if (r.mineBurnBlocks) {
+            simnet.mineEmptyBurnBlocks(r.burnBlocksToMine);
+          }
+        } catch (error: any) {
+          throw error;
```

Comment:
Don't we want to `radio.emit` the error message? Also, if this fails, do we want to keep going?

Reply:
OK, then if that fails, we basically stop (`heatstroke`). Why don't we even need to wrap in a `try-catch` then?